index.html

```diff
@@ -5,34 +5,28 @@
 	<!-- Required meta tags -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
-
-	<title>Sam, Creative Mofo</title>
-	
-	
-	
+	<title>Sam, Creative Mofo</title>	
+	<meta name="description" content="Sam's corner of the world wide web.">
```

Comment:
This shows up in some social previews so you should change it to whatever you want. its for SEO.